ee/clickhouse/models/group.py

```diff
@@ -2,12 +2,27 @@
 import json
 from typing import Dict, Optional
 
+from django.db.models.signals import post_save
+from django.dispatch.dispatcher import receiver
 from django.utils.timezone import now
 from rest_framework import serializers
 
 from ee.clickhouse.sql.groups import INSERT_GROUP_SQL
 from ee.kafka_client.client import ClickhouseProducer
 from ee.kafka_client.topics import KAFKA_GROUPS
+from posthog.models.group import Group
+from posthog.utils import is_clickhouse_enabled
+
+if is_clickhouse_enabled():
+
+    @receiver(post_save, sender=Group)
+    def person_created(sender, instance: Group, created, **kwargs):
+        create_group(
```

Comment:
mirrors the person implementation so that when groups are created from the django orm (especially during tests), the clickhouse and postgres data will match